lib/trufi_configuration.dart

```diff
@@ -92,3 +93,8 @@ class TrufiConfigurationUrl {
   var donate = "";
   var share = "";
 }
+
+class TrufiConfigurationFeatures {
```

Comment:
To make the configurations more speakable, I would recommend (also for my PR) adding documentation on top of it.

/// [showCarRouteOnTransitError] does ...